log4j-1.2-api/src/test/java/org/apache/log4j/LevelTest.java

```diff
@@ -205,6 +207,70 @@ public void testALL() {
         assertTrue(Level.ALL instanceof Level);
     }
 
+    /**
+     * Tests Level.OFF version2Level.
+     */
+    @Test
+    public void testOffVersion2Level() {
+        assertEquals(org.apache.logging.log4j.Level.OFF, Level.OFF.getVersion2Level());
+    }
+
+    /**
+     * Tests Level.FATAL version2Level.
+     */
+    @Test
+    public void testFatalVersion2Level() {
+        assertEquals(org.apache.logging.log4j.Level.FATAL, Level.FATAL.getVersion2Level());
+    }
```

Comment:
The test class uses unfortunately JUnit 4, but it should be easy to convert it to a JUnit 5 test class with a `@ParameterizedTest` instead of multiple single tests.